### PR TITLE
Added test file

### PR DESCRIPTION
**Pull request by RoMinaJin**

### CODE REVIEW

File: TEST.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<h1><a id="user-content-test" class="anchor" href="#test" aria-hidden="true"><span class="octicon octicon-link"></span></a>TEST</h1>
+
+<p>Test pull request for the class</p>
+
+<blockquote>
+<p>TODO</p>
+
+<ul>
+<li>Please ignore</li>
+<li>No really important</li>
```
Comment (richardneililagan) [inline TEST.html]: <training> `Not really important` please fix.

File: TEST.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<li>Just messing aroung</li>
```
Comment (richardneililagan) [inline TEST.html:13]: <training> `Just messing around` please fix.

File: TEST.html (hunk 1 above)
Comment (RoMinaJin) [inline TEST.html]: Noted. :dash:

**Comment by richardneililagan** [comment]
<training> Good. You can update active PR branches.

**Pull request closed without merging**